### PR TITLE
ENT-109: View Seat Entitlement on Enterprise admin screen

### PR DESCRIPTION
**Pull request by asadiqbal08**
@mattdrayer @saleem-latif FYI. Need to add tests in it to support coverage. 

[ENT-109](https://openedx.atlassian.net/browse/ENT-109)

screen shot 2017-01-20 at 3 04 21 pm
screen shot 2017-01-20 at 3 04 52 pm

### CODE REVIEW

File: enterprise/admin/__init__.py [hunk 1]
```diff
@@ -48,17 +51,58 @@ class EnterpriseCustomerIdentityProviderInline(admin.StackedInline):
...
+    extra = 0
+    can_delete = True
+    fields = ('enterprise_customer', 'entitlement_id', 'ecommerce_coupon_url',)
+
+    def ecommerce_coupon_url(self, instance):
+        """
+        Instance is EnterpriseCustomer. Return e-commerce coupon urls.
+        """
+        if not instance.entitlement_id:
+            return "N/A"
```
Comment (asadiqbal08) [inline enterprise/admin/__init__.py]: For new data.
screen shot 2017-01-20 at 6 06 38 pm

File: enterprise/admin/__init__.py [hunk 2]
```diff
@@ -48,17 +51,58 @@ class EnterpriseCustomerIdentityProviderInline(admin.StackedInline):
...
 
+class EnterpriseCustomerEntitlementInline(admin.StackedInline):
+    """
+    Django admin model for EnterpriseCustomerEntitlement.
+    The admin interface has the ability to edit models on the same page as a parent model. These are called inlines.
+    https://docs.djangoproject.com/en/1.8/ref/contrib/admin/#django.contrib.admin.StackedInline
+    """
+
+    model = EnterpriseCustomerEntitlement
+    extra = 0
```
Comment (mattdrayer) [inline enterprise/admin/__init__.py]: Sorry, what is `extra` used for?

**Comment by mattdrayer** [comment]
Just one question, otherwise LGTM!  👍 assuming no changes based on question.

File: enterprise/admin/__init__.py (hunk 2 above)
Comment (asadiqbal08) [inline enterprise/admin/__init__.py]: The number of empty forms that is displayed is controlled by the extra parameter.

**Comment by asadiqbal08** [comment]
@saleem-latif can u review it too ?

**Comment by mattdrayer** [comment]
We could have @haikuginger or @e-kolpakov take a quick look at this PR as well

File: tests/test_models.py [hunk 3]
```diff
@@ -513,3 +513,31 @@ def test_provider_name(self, mock_method):
...
+    @ddt.data(
+        str, repr
+    )
+    def test_string_conversion(self, method):
+        """
+        Test ``TestEnterpriseCustomerEntitlements`` conversion to string.
+        """
+        entitlement_id, enterprise_customer_name = 1234, "TestShib"
+        enterprise_customer = EnterpriseCustomerFactory(name=enterprise_customer_name)
+        ec_entitlements = EnterpriseCustomerEntitlement(
```
Comment (saleem-latif) [inline tests/test_models.py]: It would be better to create `EnterpriseCustomerEntitlementFactory` this would shrink these two lines into one line something like this
```
EnterpriseCustomerEntitlementFactory(
    enterprise_customer__name= enterprise_customer_name, 
    entitlement_id=entitlement_id,
)
```

File: tests/test_models.py [hunk 4]
```diff
@@ -513,3 +513,31 @@ def test_provider_name(self, mock_method):
...
+            enterprise_customer=enterprise_customer,
+            entitlement_id=entitlement_id,
+        )
+
+        expected_to_str = "<EnterpriseCustomerEntitlement {customer}: {id}>".format(
+            customer=enterprise_customer,
+            id=1234,
```
Comment (saleem-latif) [inline tests/test_models.py]: it would  be better to use `entitlement_id` instead of `1234` directly.

File: tests/test_models.py (hunk 3 above)
Comment (asadiqbal08) [inline tests/test_models.py]: @saleem-latif lines will remain the same. Not ? any other reason to use factory here ?

Comment (saleem-latif) [inline tests/test_models.py]: @asadiqbal08 It may not make much difference in this case but it would be useful to have in future.

Comment (asadiqbal08) [inline tests/test_models.py]: as per one to one discussion, we will do it another PR.

**Comment by saleem-latif** [comment]
👍

**Comment by saleem-latif** [comment]
Make sure to squash commits before merge
